### ISSUE TITLE
A failure occurred during patch application. Probably needs a rebase.

### ISSUE THREAD

**Issue by zefkerr** [Sporif/dxvk-async#62]
DXVK version: the latest git at this moment: https://github.com/doitsujin/dxvk/commit/4d0d455895fb0066dd7eba611f1dc50273f0e17f 

Hunk #2 FAILED at 908.
1 out of 2 hunks FAILED -- saving rejects to file src/dxvk/dxvk_graphics.cpp.rej
patching file src/dxvk/dxvk_graphics.h
patching file src/dxvk/dxvk_image.h
Hunk #1 succeeded at 548 (offset 1 line).
Hunk #2 succeeded at 587 (offset 1 line).
patching file src/dxvk/dxvk_options.cpp
patching file src/dxvk/dxvk_options.h
==> ERROR: A failure occurred in prepare().
Aborting...

**Comment by gabriele2000** [comment]
DXVK 2.1 is out, bring it on

**Comment by romracer** [comment]
Probably not what you're after, but maybe it'll hold you over until there's something more official in this repo:
https://github.com/romracer/dxvk/actions

**Comment by niobium93** [comment]
@romracer why not open a PR to get the updated patch on this repo?

**Comment by gabriele2000** [comment]
> Probably not what you're after, but maybe it'll hold you over until there's something more official in this repo: https://github.com/romracer/dxvk/actions

your DXVK patch is weird: DXVK 1.10 is reported in the log and it doesn't compile anything...

**Comment by romracer** [comment]
> @romracer why not open a PR to get the updated patch on this repo?

My repo is a fork of the original, not this repo.  I also didn't feel like tracking down commit hashes and what not.  Also, the change is trivial (extra argument to m_workers call).



> > Probably not what you're after, but maybe it'll hold you over until there's something more official in this repo: https://github.com/romracer/dxvk/actions
> 
> your DXVK patch is weird: DXVK 1.10 is reported in the log and it doesn't compile anything...

My patch is this patch with the extra argument to m_workers().  I'm not sure what you're talking about.  In any case, I was just suggesting to download the artifact from the GitHub Actions I have enabled (requires GitHub login I think).

My patch: https://github.com/romracer/dxvk/commit/cb9e065de68c3129289a25791f56a640a9e88791.patch
Artifact: https://github.com/romracer/dxvk/actions/runs/4007048791

**Comment by gabriele2000** [comment]
> My patch is this patch with the extra argument to m_workers(). I'm not sure what you're talking about.

I'm not sure but in the logs it gets reported at DXVK 1.10, weird... I'll try to download it again and test it...

EDIT: `
info:  Game: MW2CR.exe
info:  DXVK: v1.10.1-1113-gcb9e065d`

![image](https://user-images.githubusercontent.com/12186181/215151183-bc2ad862-23ef-4031-8c8d-3fab9628f7bf.png)
Notice the 0%, it's like that since 2 minutes.
[MW2CR_dxgi.log](https://github.com/Sporif/dxvk-async/files/10521464/MW2CR_dxgi.log)

**Comment by gabriele2000** [comment]
@Sporif any news?

**Comment by romracer** [comment]
> > My patch is this patch with the extra argument to m_workers(). I'm not sure what you're talking about.
> 
> I'm not sure but in the logs it gets reported at DXVK 1.10, weird... I'll try to download it again and test it...
> 
> EDIT: ` info: Game: MW2CR.exe info: DXVK: v1.10.1-1113-gcb9e065d`
> 
> ![image](https://user-images.githubusercontent.com/12186181/215151183-bc2ad862-23ef-4031-8c8d-3fab9628f7bf.png) Notice the 0%, it's like that since 2 minutes. [MW2CR_dxgi.log](https://github.com/Sporif/dxvk-async/files/10521464/MW2CR_dxgi.log)

I think the log says v1.10.1 because of me force-pushing and default brances, etc. when those actions are triggered.  The commit hash ("cb9e065d") matches the commit I referenced before.  In any case, I've uploaded a pre-release asset that logs the correct version.

https://github.com/romracer/dxvk/releases

If it still doesn't work, the async patch may need further adjustments.

**Comment by gabriele2000** [comment]
> If it still doesn't work, the async patch may need further adjustments.

Yup, it needs adjustements because disabling async will fix the problem.
